crates/wasmtime/src/runtime/store.rs

```diff
@@ -1203,6 +1219,28 @@ impl<T> Store<T> {
     pub fn debug_frames(&mut self) -> Option<crate::DebugFrameCursor<'_, T>> {
         self.as_context_mut().debug_frames()
     }
+
+    /// Set the debug callback on this store.
+    ///
+    /// See [`crate::DebugHandler`] for more documentation.
+    #[cfg(feature = "debug")]
+    pub fn set_debug_handler(&mut self, handler: impl DebugHandler<Data = T>)
+    where
+        T: Send,
```

Comment:
This is a subtle, but load bearing, `where` clause. Could the function have a Rust-level comment (e.g. `//`, not `///`) indicating why? Basically explaining how a `&mut dyn VMStore` is gonna be on the stack and that is required to be `Send`, so therefore this must be `Send`.

Reply:
Yep definitely -- added in 50bb08e40e.